### ISSUE TITLE
Standard Library Doesn't return connected Algorand public address

### ISSUE THREAD

**Issue by KBryan** [reach-sh/reach-lang#128]
When reviewing a video. I noticed that the return address starts with 0x this would assume a public Ethereum address. As you can see from the example the standard library should return an Algorand address.

![Screen Shot 2021-04-30 at 11 21 38 AM](https://user-images.githubusercontent.com/681817/116717666-24fb1780-a9a7-11eb-8b79-0cb7ffec0179.png)

**Comment by jeapostrophe** [comment]
Thanks @KBryan . I've assigned @DanBurton to work on "Implement formatAddress to turn a NetworkAccount into the way accounts are formatted for the given network".

In Reach's JavaScript support --- `Address` is represented by an internal `NetworkAccount` (see: https://docs.reach.sh/ref-frontends-js-types.html )  which on Algorand is just the public key, which you see displayed there. However, it is typical for people to not think of their public key as their address, but instead to write out the base32 encoding of the public key AND a checksum. You can see this in this function of the Algorand JS SDK --- https://github.com/algorand/js-algorand-sdk/blob/fd187cedab70cb123333ba62ca3e6fe98c1e7d3d/src/encoding/address.ts#L94-L110

Basically, we will make it so that Reach has a way to "format for user consumption" an address, just like we have `formatCurrency`.

Thanks again!

**Comment by KBryan** [comment]
Sounds good, will keep this issue open until the solution mentioned has been implemented and tested.

**Comment by DanBurton** [comment]
There is a new stdlib function, `formatAddress`, that is available for this. It accepts an `Account`, `NetworkAccount`, or an `Address`, and formats it in the network-specific way users expect to see it.

https://docs.reach.sh/ref-frontends-js-utils.html#%28javascript._%28%28format.Address%29%29%29

Available as of `@reach-sh/stdlib@0.1.2-rc.46`

**Comment by KBryan** [comment]
Ok, are there any test cases for this?

**Comment by DanBurton** [comment]
If you're curious:
https://github.com/reach-sh/reach-lang/commit/279dcbde6ae5c112d2e8f35bfe031884aeeef8d0#diff-238f2c0082bb61188656d618a2e0069050a4512be69c42a0fd3b798ce7a42042
